app/code/Magento/Tax/view/frontend/web/js/view/checkout/summary/tax.js

```diff
@@ -13,13 +13,17 @@ define([
     'Magento_Checkout/js/model/quote',
     'Magento_Checkout/js/model/totals',
     'jquery',
-    'mage/translate'
-], function (ko, Component, quote, totals, $, $t) {
+    'mage/translate',
+    'underscore'
+], function (ko, Component, quote, totals, $t, _) {
     'use strict';
 
     var isTaxDisplayedInGrandTotal = window.checkoutConfig.includeTaxInGrandTotal,
         isFullTaxSummaryDisplayed = window.checkoutConfig.isFullTaxSummaryDisplayed,
         isZeroTaxDisplayed = window.checkoutConfig.isZeroTaxDisplayed;
+        totalPercentage = 0,
```

Comment:
Comma-separated variables declaration continues after a semicolon, it should be replaced with a comma.

Reply:
Hi @sivaschenko I've updated code, next time I will do with cherry-pick. Thanks :+1: